lib/absinthe/phase/telemetry.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Absinthe.Phase.Telemetry do
+  @moduledoc """
+  Gather and report telemetry about a pipeline.
+  """
+  @telemetry_event [:absinthe, :query]
```

Comment:
@benwilson512 @bruce 

Wanted to double-check with you on naming.. I went back and forth a little on this. This represents the `pipeline`, but there's also now a schema pipeline...

Reply:
if we're going with event sounding names then probably `[:absinthe, :executed]`

Reply:
Other examples:

* Plug: `[:my, :plug, :start]` / `[:my, :plug, :stop]`
* Ecto: `[:repo, :query]`
* Redix: `[:redix, :pipeline]` (redis pipelined commands)

Reply:
Hm. Well I definitely want to avoid `:query` because in a graphql context that word has specific meaning. `[:absinthe, :exec]` seems to fit the existing styles maybe?

Reply:
My question was more about trying to align the language to the exact details of what Absinthe is doing (execute pipeline) vs what users understand to be happening (query)

I just looked at the language in the GraphQL spec..  https://graphql.github.io/graphql-spec/draft/#sec-Executing-Requests  what about:

`[:absinthe, :execute, :request]`

Reply:
Sorry, wasn't following. `[:absinthe, :execute, :operation]` perhaps? Request sounds too web focused to me.

Reply:
👍